README.md

```diff
@@ -485,6 +485,22 @@ Rodauth::Rails.configure do |config|
 end
 ```
 
+## Working with JWT
+
+To work with JWT, you'll need to enable json in `Roda`, and the [JWT plugin][rodauth_jwt_doc]
+
+[rodauth_jwt_doc]: https://github.com/jeremyevans/rodauth/blob/master/doc/jwt.rdoc
```

Comment:
If you could just move this to the bottom of the document with the other links, I will merge the PR.

Reply:
Also, we should link to the RDoc-generated version that you can find on the Rodauth website, as that one will have syntax highlighting in case of any code blocks.

Reply:
Done :)